content/docs/guides/responsive_amp@es.md

```diff
@@ -60,13 +36,14 @@ Por ejemplo:
         background-color: grey;
       }
     </style>
-    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    ...
   </head>
 [/sourcecode]
 
-**Importante:**
+{% call callout('Important', type='caution') %}
```

Comment:
Just that first text - it's the word that displays in the callout like:
![selection_005](https://cloud.githubusercontent.com/assets/1012254/26557841/5b3eaae4-4472-11e7-853b-54253721fbf7.png)